invenio_app_rdm/config.py

```diff
@@ -671,6 +677,7 @@ def files_rest_permission_factory(obj, action):
     **NAMES_TRANSFORMERS,
     **FUNDERS_TRANSFORMERS,
     **AWARDS_TRANSFORMERS,
+    **AFFILIATIONS_TRANSFORMERS,
```

Comment:
Would it make sense to replace the duplicated aliased imports `FUNDERS_TRANSFORMERS` and `AFFILIATIONS_TRANSFORMERS` by only one import named `COMMON_ROR_TRANSFORMERS`?

I think you'll need a round of `isort` too.